tripy/nvtripy/frontend/module/groupnorm.py

```diff
@@ -104,6 +113,10 @@ def __init__(
         self.bias = DefaultParameter((num_channels,), dtype=dtype)
         self.eps = eps
 
+        self._instance_norm = InstanceNorm(num_groups, dtype=dtype, eps=eps)
+        self._instance_norm.weight = ones((num_groups,), dtype=dtype)
+        self._instance_norm.bias = zeros((num_groups,), dtype=dtype)
```

Comment:
Can we just call the `instancenorm` function? I don't think this way will work out well with `state_dict`s because there would be extra keys.